src/card/card-media.jsx

```diff
@@ -90,14 +90,14 @@ const CardMedia = React.createClass({
 
   render() {
     let styles = this.getStyles();
-    let rootStyle = this.prepareStyles(styles.root, this.props.style);
-    let mediaStyle = this.prepareStyles(styles.media, this.props.mediaStyle);
-    let overlayContainerStyle = this.prepareStyles(styles.overlayContainer, this.props.overlayContainerStyle);
-    let overlayContentStyle = this.prepareStyles(styles.overlayContent, this.props.overlayContentStyle);
-    let overlayStyle = this.prepareStyles(styles.overlay, this.props.overlayStyle);
+    let rootStyle = this.mergeStyles(styles.root, this.props.style);
+    let mediaStyle = this.mergeStyles(styles.media, this.props.mediaStyle);
+    let overlayContainerStyle = this.mergeStyles(styles.overlayContainer, this.props.overlayContainerStyle);
+    let overlayContentStyle = this.mergeStyles(styles.overlayContent, this.props.overlayContentStyle);
+    let overlayStyle = this.mergeStyles(styles.overlay, this.props.overlayStyle);
 
     let children = React.Children.map(this.props.children, (child) => {
-      return React.cloneElement(child, {style: this.prepareStyles(styles.mediaChild, child.props.style)});
+      return React.cloneElement(child, {style: this.mergeStyles(styles.mediaChild, child.props.style)});
```

Comment:
`CardMedia` doesn't seem to take Material Ui component. Shouldn't we use `prepareStyles`?